src/modules/i18n/reducers/contentLanguageReducer.js

```diff
@@ -1,14 +1,10 @@
 // @flow
 
 import type { SetContentLanguageActionType } from '../../app/StoreActionType'
-import { handleAction, type ReduxReducer } from 'redux-actions'
 import { DEFAULT_LANGUAGE } from '../components/I18nProvider'
 
-const contentLanguageReducer: ReduxReducer<string, SetContentLanguageActionType> =
-  handleAction(
-    'SET_CONTENT_LANGUAGE',
-    (state: string, action: SetContentLanguageActionType) => action.params.contentLanguage,
-    DEFAULT_LANGUAGE
-  )
+const contentLanguageReducer = (state: string = DEFAULT_LANGUAGE, action: SetContentLanguageActionType): string => {
```

Comment:
The type of this reducer is false. action should be of type `StoreActionType`. (Then also the reducer body must be adjusted)